bookdown/01-mlr3-basics.Rmd

```diff
@@ -1,28 +1,31 @@
 # mlr3 Basics {#basics}
 
-This chapter will teach you the essential building blocks, R6 classes and operations of `r mlr_pkg("mlr3")`.
-These include creating supervised ML tasks like classification/regression, performing training/prediction and cross-validate/benchmark different models.
+This chapter will teach you the essential building blocks, R6 classes, and operations of `r mlr_pkg("mlr3")`.
+This includes creating supervised machine learning tasks like classification and regression, training models and getting prediction on new data, and evaluating and comparison of different models through cross-validation and benchmarking.
 
-A typical (simple) machine-learning workflow looks like this:
+A typical machine learning workflow looks like this:
```

Comment:
Should we make a global decision how we want ML to be written? I.e. either

- machine-learning
- machine learning

Reply:
in english you do not use hyphens for this.... so pls "machine learning" or even ML